strawberry/django/test/client.py

```diff
@@ -0,0 +1,66 @@
+import json
+from typing import Any, Dict, Optional
+
+from typing_extensions import Literal
+
+from strawberry.test import BaseGraphQLTestClient, Response
+
+
+class GraphQLTestClient(BaseGraphQLTestClient):
```

Comment:
Do we want to name this specific to Django? Or will each integration's client have the same name here?

Reply:
I didn't check how to do with the other integrations... If possible I would keep the same parameters/features...